internal/ackhandler/sent_packet_handler_test.go

```diff
@@ -944,6 +944,18 @@ var _ = Describe("SentPacketHandler", func() {
 		})
 	})
 
+	Context("amplification limit, for the server, with validated address", func() {
+		JustBeforeEach(func() {
+			rttStats := utils.NewRTTStats()
+			handler = newSentPacketHandler(42, protocol.InitialPacketSizeIPv4, rttStats, true, perspective, nil, utils.DefaultLogger)
+		})
+
+		It("do not limits the window", func() {
+			Expect(handler.SendMode()).To(Equal(SendAny))
+			Expect(handler.peerAddressValidated).To(Equal(true))
```

Comment:
It would be better to test the expected behavior than to make assertion on implementation details.